pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -100,15 +102,34 @@ func setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, uid stri
 		return nil
 	}
 
+	cache := PodCacheInterface{Iface: iface}
+	podIPv6 := ""
+	indexIPv6 := 0
```

Comment:
I feel instead of this, it would be more cleaner to have a `PodIPv6` field in `PodCacheInterface`

Reply:
i started with that, but figured out that the current approach let just one place to put ipv6 always at the end of ips, and ipv4 in the beginning + in the podip , because this is the only supported options atm by k8s due to CRI bug.
else we would need to add conditions where we read the file, and support cases where there is just ipv4 just ipv6 and so on, where this method support them all in one shot

Reply:
done